src/Columns/Column.php

```diff
@@ -167,14 +175,14 @@ protected function renderFooterCellContent()
      * Renders the data cell content.
      *
      * @param mixed $model the data model
-     * @param mixed $key   the key associated with the data model
-     * @param int   $index the zero-based index of the data model among the models array returned by [[GridView::dataProvider]].
-     *
+     * @param mixed $key the key associated with the data model
+     * @param int $index the zero-based index of the data model among the models array returned by
+     *     [[GridView::dataProvider]].
      * @return string the rendering result
      */
-    protected function renderDataCellContent($model, $key, $index)
+    protected function renderDataCellContent($model, $key, $index): string
     {
-        if ($this->content !== null) {
+        if (is_callable($this->content)) {
```

Comment:
Why did you changed that? It is less efficient and hides configuration errors (invalid callable will be just ignored).

Reply:
I thought that `content` has 2 types: a callable and a string.

Reply:
That is even worse, because string can be callable, so literal content could be interpreted as function name. It should be either always callable or only `Closure` should be supported.

Reply:
`$content` in `Column` is `callable` or null. It's ok.